utils.ts

```diff
@@ -9,6 +9,28 @@ import shelljs from "shelljs";
 import { createServer } from "vite";
 import chalk from "chalk";
 import { generate } from "./generate";
+import axios, { Method } from "axios";
+import { Stream } from "stream";
```

Comment:
since this is a node stdlib import, can you put it up next to it others (like right below `crypto`)?

Reply:
i'm going to do the import sorting through linting soon